workers/unity/Assets/Gdk/Physics/Components/BufferedTransform.cs

```diff
@@ -6,6 +6,7 @@ namespace Improbable.Gdk.TransformSynchronization
     public class BufferedTransform : Component
     {
         public System.Collections.Generic.List<SpatialOSTransform> TransformUpdates;
+        public SpatialOSTransform LastTransformSnapshot;
```

Comment:
We are not using this variable anywhere. Only updating its content whenever we have a new update

Reply:
This is needed in `ApplyTransformUpdatesSystem` because we now need to reconstruct component snapshots from the updates since we don't get the component snapshots directly anymore.